docs/testing/debug-with-vs-extension.md

```diff
@@ -0,0 +1,73 @@
+
+![VS Code Debugger Extension debugging a section of Excel Add-ins](../images/vs-debugger-extension-for-office-addins.jpg)
+
+## Pre-Requisites
+
+- [Visual Studio Code](https://code.visualstudio.com/) (must be run as an administrator)
+- [Node.js (version 10+)](https://nodejs.org/)
+- Windows 10
+- [Microsoft Edge](https://www.microsoft.com/edge)
+- [Yo Office](https://github.com/OfficeDev/generator-office)
+
+These instructions assume you have experience using the command line, understand basic JavaScript, and have created an Office add-in project before using the Yo Office generator. If you haven't done this before, consider visiting one of our tutorials, like this [Excel Office Add-in tutorial](../tutorials/excel-tutorial.md).
+
+## Install and use
+
+1. [Create a new project with the latest version of Yo Office](https://docs.microsoft.com/office/dev/add-ins/quickstarts/excel-quickstart-jquery?tabs=yeomangenerator). Follow the prompts within the command line to set up your project. You can choose any language or type of project to suit your needs.
+
+2. Open a command prompt as administrator
+   ![Command prompt options, including "run as administrator" in Windows 10](../images/run-as-administrator-vs-code.jpg)
+
+3. Navigate to your project directory.
+
+4. Run the following command to open your project in Visual Studio Code as an administrator.
+
+```command line
+code .
+```
+
+Once Visual Studio Code is open, navigate manually to the project folder.
+
+> [!TIP]
+> To open Visual Studio Code as an administrator, select the **run as administrator** option when opening Visual Studio Code from the Start Menu in Windows.
+
+5. Within VS Code, select **CTRL + SHIFT + X** to open the Extensions bar. Search for the "VS Code Debugger for Office Add-ins" extension and install it.
+
+6. In the .vscode folder of your project, you will find a launch.json file. At the end of the file, add the following code to the configurations section of the file.
+
+```JSON
+{
+  "type": "office-addin",
+  "request": "attach",
+  "name": "Attach to Office Add-ins",
+  "port": 9222,
+  "trace": "verbose",
+  "url": "https://localhost:3000/taskpane.html?_host_Info=HOST$Win32$16.01$en-US$$$$0",
+  "webRoot": "${workspaceFolder}",
+  "timeout": 45000
+}
+```
+
+7. In the section of JSON you just copied, find the "url" section. In this URL, you will need to replace the uppercase HOST text with the host application for your Office add-in. For example, if your Office add-in is for Excel, your URL value would be "https://localhost:3000/taskpane.html?_host_Info=<strong>Excel</strong>$Win32$16.01$en-US$\$\$\$0".
+
+8. Open the command prompt and ensure you are at the root folder of your project. Run the command `npm start` to start the dev server. When your add-in loads in Excel, open the task pane.
```

Comment:
Consider removing the reference to Excel, since that was just the example in the last section.
```suggestion
8. Open the command prompt and ensure you are at the root folder of your project. Run the command `npm start` to start the dev server. When your add-in loads in the Office client, open the task pane.
```